### PR TITLE
chore(build): replaced `Karma` with `Jest`

### PR DESCRIPTION
**Pull request by claylaut**

### CODE REVIEW

File: build/tasks/test.js [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
-	runTests(true, cb);
-});
+const jestConfig = {
+	rootDir: 'source'
+};
 
-gulp.task("tdd", ["compile:test"], (cb) => {
-	runTests(false, cb);
+gulp.task("test", ["compile:test"], (cb) => {
+	return runSeq(
```
Comment (alan-agius4) [inline build/tasks/test.js]: Formatting looks alward 😝

File: README.md [hunk 2]
```diff
@@ -69,15 +69,15 @@ aurelia.use.plugin("@ssv/au-core", config);
...
 - Visual Studio Code or similar code editor
-- TypeScript 2.2+
+- TypeScript 2.5+
 - Git + SourceTree, SmartGit or similar (optional)
 - Ensure to install **global NPM modules** using the following:
 
 
 ```bash
-npm install -g git gulp yarn karma-cli
+npm install -g git gulp yarn jest-cli
```
Comment (stephenlautier) [inline README.md]: is it necessary `jest-cli` to be installed globally?

File: build/tasks/test.js [hunk 3]
```diff
@@ -1,38 +1,24 @@
...
-		// make sure failed karma tests cause gulp to exit non-zero
-		if (code === 1) {
-			gutil.log(gutil.colors.red("------- Error: unit test failed -------"));
-			return process.exit(1);
-		}
-		cb();
-	}).start();
-}
+gulp.task("jest", () => {
+	jest.runCLI({ config: jestConfig }, ".", () => done());
```
Comment (stephenlautier) [inline build/tasks/test.js]: you are invoking `done` but its not passed as a param

also can be written like
```js
gulp.task("jest", cb => jest.runCLI({ config: jestConfig }, ".", cb));
```

File: README.md (hunk 2 above)
Comment (claylaut) [reply README.md]: not really it's not required. I will remove it

File: build/tasks/test.js (hunk 3 above)
Comment (claylaut) [reply build/tasks/test.js]: done

File: build/tasks/test.js (hunk 1 above)
Comment (claylaut) [reply build/tasks/test.js]: the formatting is correct :)

File: README.md (hunk 2 above)
Comment (claylaut) [reply README.md]: updated, removed jest-cli from readme

**Pull request merged**